static/locales/en.client.json

```
@@ -1,18 +1,763 @@
+    "InvalidDocTourBody": "Cannot construct a document tour from the data in this document. Ensure there is a table named GristDocTour with columns Title, Body, Placement, and Location."
+  },
+  "DocumentSettings": {
+    "DocumentSettings": "Document Settings",
+    "ThisDocumentID": "This document's ID (for API use):",
+    "TimeZone": "Time Zone:",
+    "Locale": "Locale:",
+    "Currency": "Currency:",
+    "LocalCurrency": "Local currency ({{currency}})",
+    "EngineRisk": "Engine (experimental {{span}} change at own risk):",
+    "Save": "Save",
+    "SaveAndReload": "Save and Reload"
+  },
+  "DuplicateTable": {
+    "NewName": "Name for new table",
+    "AdviceWithLink": "Instead of duplicating tables, it's usually better to segment data using linked views. {{link}}",
+    "CopyAllData": "Copy all data in addition to the table structure.",
+    "WarningACL": "Only the document default access rules will apply to the copy."
+  },
+  "errorPages": {
+    "AccessDenied": "Access denied{{suffix}}",
+    "DeniedOrganizationDocuments": "You do not have access to this organization's documents.",
+    "SignInWithDifferentAccount": "You are signed in as {{email}}. You can sign in with a different account, or ask an administrator for access.",
+    "SignInToAccess": "Sign in to access this organization's documents.",
+    "GoToMainPage": "Go to main page",
+    "SignIn": "Sign in",
+    "AddAcount": "Add account",
+    "SignedOut": "Signed out{{suffix}}",
+    "GenericError": "Error{{suffix}}",
+    "SignedOutNow": "You are now signed out.",
+    "SignedInAgain": "Sign in again",
+    "PageNotFound": "Page not found{{suffix}}",
+    "NotFoundMainText": "The requested page could not be found.{{separator}}Please check the URL and try again.",
+    "ContactSupport": "Contact support",
+    "SomethingWentWrong": "Something went wrong",
+    "ErrorHappened_message": "There was an error: {{message}}",
+    "ErrorHappened_unknown": "There was an unknown error."
+  },
+  "FieldConfig": {
+    "ColumnLabel": "COLUMN LABEL AND ID",
+    "ColumnOptionsLimited": "Column options are limited in summary tables.",
+    "ColumnType_formula_one": "Formula Column",
+    "ColumnType_data_one": "Data Column",
+    "ColumnType_empty_one": "Empty Column",
+    "ColumnType_formula_other": "Formula Columns",
+    "ColumnType_data_other": "Data Columns",
+    "ColumnType_empty_other": "Empty Columns",
+    "ColumnType_mixed_other": "Mixed Behavior",
+    "ConvertColumn_formula": "Clear and make into formula",
+    "ConvertColumn_data": "Convert column to data",
+    "ConvertColumn_triggerformula": "Convert to trigger formula",
+    "ClearAndReset": "Clear and reset",
+    "EnterFormula": "Enter formula",
+    "ColumnBehavior": "COLUMN BEHAVIOR",
+    "SetFormula": "Set formula",
+    "SetTriggerFormula": "Set trigger formula",
+    "MakeIntoDataColumn": "Make into data column",
+    "TriggerFormula": "TRIGGER FORMULA"
+  },
+  "ExampleInfo": {
```

```suggestion
  "ExampleCard": {
```

This is tied to the other comment about moving translation to ExampleCard.ts.